contribs/gmf/src/directives/print.js

```diff
@@ -109,11 +125,14 @@ gmf.printDirective = function(gmfPrintTemplateUrl) {
     templateUrl: gmfPrintTemplateUrl,
     replace: true,
     restrict: 'E',
+    transclude: true,
     scope: {
       'map': '=gmfPrintMap',
       'active': '=gmfPrintActive',
       'rotateMask': '&?gmfPrintRotatemask',
-      'fieldValues': '&?gmfPrintFieldvalues'
+      'fieldValues': '&?gmfPrintFieldvalues',
+      'hiddenAttributeNames': '=gmfPrintHiddenattributes',
```

Comment:
Are the 2 new fields mandatory ? If not use `?`, otherwise it breaks all code that currently use the directive.

Also, is the 2 ways binding usefull ? if not use the `<`, it seems not as you pass expression in the attribute value like `gmf-print-hiddenattributes="['title', 'comments']"`.

@sbrunner @fredj @gberaudo @ger-benjamin 
Please try to more strict with your bindings.